static/styles/settings.css

```diff
@@ -520,19 +520,20 @@ input[type="checkbox"] {
 }
 
 .alert-notification {
-    display: inline-block;
-    vertical-align: top;
-    height: auto !important;
-    width: auto !important;
-
-    background-color: transparent;
-    border-radius: 4px;
-    margin-top: 14px;
-    margin-left: 10px;
-    color: hsl(156, 30%, 50%);
-    padding: 3px 10px;
+    display: none;
```

Comment:
Can you add a comment repeating some of the commit message notes here?  And maybe also show a screenshot of what the border looks like; we may want to backport this commit to 4.x if it's really ugly.

Reply:
On checking for screenshot, I cannot reproduce this. But I remember seeing empty alert box, after adding a basic tabbed navigation widget for trial. I have dropped this commit now but I will add it again if it appears again after adding tabbed navigation.

Reply:
OK.